bazel/README.md

```diff
@@ -53,9 +54,10 @@ Envoy compiles and passes tests with the version of clang installed by XCode 8.3
 Apple LLVM version 8.1.0 (clang-802.0.42).
 
 3. Install Golang on your machine. This is required as part of building [BoringSSL](https://boringssl.googlesource.com/boringssl/+/HEAD/BUILDING.md)
-and also for [Buildifer](https://github.com/bazelbuild/buildtools) which is used for formatting bazel BUILD files.
-4. `bazel fetch //source/...` to fetch and build all external dependencies. This may take some time.
-5. `bazel build //source/exe:envoy-static` from the Envoy source directory.
+and also for [Buildifer](https://github.com/bazelbuild/buildtools) which is used for formatting bazel BUILD files. 
+4. `go get github.com/bazelbuild/buildtools/buildifier` to install buildifier
```

Comment:
Why only on Apple?

Reply:
Should we split the OS stuff out into a unique heading?

Reply:
Never mind, I was reading this incorrectly via the GH diffs, looks good.